app/models/open_weather_map.rb

```diff
@@ -0,0 +1,47 @@
+COOLDOWN_IN_MINUTES = 10
+OPEN_WEATHER_MAP_URI = 'http://api.openweathermap.org/'
+OPEN_WEATHER_MAP_PATH = 'data/2.5/forecast/daily'
+OPEN_WEATHER_MAP_CRITERIA = 'zip=22630,us'
+OPEN_WEATHER_MAP_KEY = '3fcb4e23443750cc1256f3016a160ca7'
```

Comment:
I understand. I didn't care about this one, since it's a throw-away key, but I get that's it's a bad habit I should avoid forming. I'll move it to the secrets.yml file and have git ignore that.